packages/dev-env/src/util.ts

```diff
@@ -1,13 +1,22 @@
-import { DidResolver, HandleResolver } from '@atproto/identity'
+import { DidResolver, HandleResolver, IdResolver } from '@atproto/identity'
 import { TestPds } from './pds'
+import { TestBsky } from './bsky'
 
-export const mockNetworkUtilities = (pds: TestPds) => {
+export const mockNetworkUtilities = async (pds: TestPds, bsky?: TestBsky) => {
```

Comment:
this change snuck in here because I used the approach of mocking the function on the instance (instead of the prototype) in the handle invalidation test

Do you see any reason why this approach doesn't work?

Reply:
Seems like it should be fine 👍